libcxx/test/std/utilities/format/format.range/format.range.fmtset/format.functions.tests.h

```diff
@@ -156,7 +157,7 @@ void test_char_string(TestFunction check, [[maybe_unused]] ExceptionTest check_e
 
   check_exception("The format string contains an invalid escape sequence", SV("{:}<s}"), input);
   check_exception("The fill option contains an invalid value", SV("{:{<s}"), input);
-  check_exception("The fill option contains an invalid value", SV("{::<s}"), input);
+  check_exception("The type option contains an invalid value for a character formatting argument", SV("{::<s}"), input);
```

Comment:
Can you move this test to the "type" testing part (line 181 in the original code).